README.md

```diff
@@ -242,6 +242,17 @@ The included `.vscode` automatically connects to the webpack development server
 # Types
 > When you include a module that doesn't include Type Definitions inside of the module you can include external Type Definitions with @types
 
+i.e, to have youtube api support, run this command in terminal: 
+```shell
+npm i @types/youtube @types/gapi @types/gapi.youtube
+``` 
+Afterwards, add these to **"src/custom-typings.d.ts"**: 
```

Comment:
so I would mention that you only need to do this for certain editors that doesn't support typescript 2.x or if the @types were not listed in the types field inside `tsconfig`